### ISSUE TITLE
Document key-priorieties for development

### ISSUE THREAD

**Issue by 6543** [woodpecker-ci/woodpecker#1884]
1. provide sensible defaults
2. keep required configuration minimal

---
https://github.com/woodpecker-ci/woodpecker/issues/1860#issuecomment-1590175406

**Comment by anbraten** [comment]
I disagree with the second point at least with its priority. In case of configs its more important to have explaining configs instead of "short" configs as for example DRY is a programming concept, but repeating in configs is fine and most of the time more readable IMO.

**Comment by 6543** [comment]
Good point
